src/main/drivers/sonar_hcsr04.c

```diff
@@ -75,9 +75,13 @@ void EXTI9_5_IRQHandler(void)
 }
 #endif
 
-void hcsr04_init(const sonarHardware_t *initialSonarHardware, sonarRange_t *sonarRange)
+void hcsr04_set_sonar_hardware(const sonarHardware_t *initialSonarHardware)
```

Comment:
Is there some reason for separating initialSonarHardware step? sonarHardware is latched in init and can't be changed later